build/variables/tests.yml

```diff
@@ -2,4 +2,5 @@ variables:
   LogicApps.ClientId: '60b83144-d508-4845-996f-756949ed2b21'
   LogicApps.ResourceGroup: 'Invictus-tst-testing'
   LogicApps.TestLogicAppName: 'TestLogicApp'
-  LogicApps.TestLogicAppMockingName: 'TestLogicAppMocking'
\ No newline at end of file
+  LogicApps.TestLogicAppMockingName: 'TestLogicAppMocking'
+  LogicApps.TestUpdateLogicAppName: 'invictus-tst-we-rcv-trigger-http'
```

Comment:
Perhaps we should rename this, as it's being used for both the `Update` and `Run`-methods.
I'd like to review this compared to the current TestLogicApp and have it replace that one eventually, because there isn't that much of a difference between those 2.
Should be set the name to something like: `TestBaseLogicAppName`?

Reply:
Yes, of course. `TestBaseLogicAppName` is good for me. We can always review later once we have reviewed the logic apps (renamed/removed/...).

Reply:
Done.